src/Behat/Context.php

```diff
@@ -45,7 +45,9 @@ public function waitUntilLoadingAndAnimationFinished(AfterStepScope $event): voi
     {
         $this->jqueryWait();
         $this->disableAnimations();
-        $this->assertNoException();
+        if ($event->getFeature()->getTitle() !== 'Exception') {
```

Comment:
we need to have a special step for testing exceptions, this is a dirty hack, I will fix it